CMakeLists.txt

```diff
@@ -338,6 +339,12 @@ if(HAVE_SYSINFO_H)
     target_compile_definitions(libfastfetch PUBLIC FF_HAVE_SYSINFO_H)
 endif()
 
+CHECK_INCLUDE_FILE("utmpx.h" HAVE_UTMPX_H)
+if(HAVE_UTMPX_H)
+    # needs to be public, because changes fastfech.h ABI
+    target_compile_definitions(libfastfetch PUBLIC FF_HAVE_UTMPX_H)
```

Comment:
This can be `PRIVATE` because it isn't used in a header included by `fastfetch.c` / `flashfetch.c`.